search/create_datastore_and_search.ipynb

```diff
@@ -0,0 +1,3120 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        data_store.content_config = "CONTENT_REQUIRED"
</pre><p>Initialize these parameters when creating the <code>DataStore</code> object </p><p> </p><pre class="ql-syntax" spellcheck="false">data_store = discoveryengine_v1alpha.DataStore(
  display_name=data_store_name,
  industry_vertical="GENERIC",
  content_config="CONTENT_REQUIRED"
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/271/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>